model/src/pyrenew/process/rtrandomwalk.py

```diff
@@ -24,35 +24,27 @@ class RtRandomWalkProcess(RandomVariable):
 
     def __init__(
         self,
-        Rt0_dist: dist.Distribution = dist.TruncatedNormal(
-            loc=1.2, scale=0.2, low=0
-        ),
-        Rt_transform: t.Transform | None = None,
-        Rt_rw_dist: dist.Distribution = dist.Normal(0, 0.025),
+        Rt0_dist: dist.Distribution,
+        Rt_transform: t.Transform,
+        Rt_rw_dist: dist.Distribution,
```

Comment:
```suggestion
        Rt0_dist: dist.Distribution,
        Rt_rw_dist: dist.Distribution,
        Rt_transform: t.Transform | None,
```
reorder because `Rt0_dist` and `Rt_rw_dist` are required and `Rt_transform` is optional.
If `Rt_transform is None` make it the `IdentityTransform`

Reply:
Addressed in ce3ef74.